docs/DUAL_BAND.md

```diff
@@ -2,17 +2,19 @@
 
 ([back to main page](../README.md))
 
-**Important: Dual band functionality is experimental and currently only available in the branch 'dev-dualband2'. Firmware binaries need to be compiled manually for the specific hardware.**
-
 mLRS allows for a dual-band setup, where the Tx module and receiver both transmit and receive in the 2.4 GHz and 868/915 MHz (or 433 MHz/70 cm) bands simultaneously.
 
-This operation mode is very simlar to a full-diversity setup, except that the hardware of course needs to provide two RF chains operating in the two different RF bands. An example is given by the dual-band version of the easy-to-solder boards, which can be eqipped with E77 modules working at 868/915 MHz and E28 modules working at 2.4 GHz. See [here]( https://github.com/olliw42/mLRS-hardware/tree/master/olliw-stm32-based/rx-tx-E77-E28-dualband-easysolder). Similar to a full-diversity setup, dual band uses FHSS and the receiver will choose the packet on the best band.
+Operation is similar to a full-diversity setup, except that the hardware uses two RF stages operating in the two different RF bands. Two examples are provided:
+- To use 868/915 MHz and 2.4 GHz simultaneously there is a dual-band version of the E77 Easy-Solder board, which has an E77 module working at 868/915 MHz and an E28 module working at 2.4 GHz. More details can be found [here](https://github.com/olliw42/mLRS-hardware/tree/master/olliw-stm32-based/rx-tx-E77-E28-dualband-easysolder).
+- To use 868/915 MHz and 433 MHz simultaneously one can use the "normal" E77 Easy-Solder board, when equipped with an E77 module working at 868/915 MHz and an E22 module working at 433 MHz.
```

Comment:
Small tweak:

'To use 868/915 MHz and 433 MHz simultaneously one can use the "normal" E77 Easy-Solder board equipped with an E77 module working at 868/915 MHz and an E22 module working at 433 MHz'

Reply:
THX. Done.